src/go/rpk/pkg/cli/cluster/storage/list-mountable.go

```diff
@@ -0,0 +1,85 @@
+package storage
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/redpanda-data/common-go/rpadmin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/adminapi"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newListMountable(fs afero.Fs, p *config.Params) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list-mountable",
+		Short: "List mountable topics from cloud storage",
```

Comment:
```suggestion
		Short: "List mountable topics from object storage",
```
We've generally standardized to `object storage` in the docs, especially in the context of [Tiered Storage](https://docs.redpanda.com/current/manage/tiered-storage/)